CMakeLists.txt

```diff
@@ -126,6 +126,18 @@ OPTION( KWIVER_ENABLE_C_BINDINGS               "Enable C bindings libraries" OFF
 CMAKE_DEPENDENT_OPTION( KWIVER_ENABLE_PYTHON   "Enable python code" OFF
   KWIVER_ENABLE_C_BINDINGS OFF )
 
+if (WIN32)
+  # Set default for python tests to OFF on windows for now
+  CMAKE_DEPENDENT_OPTION( KWIVER_ENABLE_PYTHON_TESTS "Enable python tests" OFF
+    "KWIVER_ENABLE_PYTHON; KWIVER_ENABLE_TESTS" OFF )
```

Comment:
This `;` may need to be `AND` instead since I think this is a logical expression that need to be evaluated.  I'm not sure what happens with the `;`.  Maybe it only evaluates or the other?

Reply:
The CMake documentation suggests that the `;` is sufficient https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html

Reply:
Interesting.  Okay, as long as that is supported I have not objections.